docker-compose.yml

```
@@ -82,6 +90,7 @@ services:
     #   - traefik.http.routers.dashboard.service=api@internal
     #   - traefik.http.routers.dashboard.rule=Host(`${CVAT_HOST:-localhost}`)
     ports:
+      - 80:80
       - 8080:8080
```

[Store config in the environment](https://12factor.net/config)

```suggestion
      - ${WEB_PORT:-8080}:8080
```